doc/README.md

```diff
@@ -0,0 +1,177 @@
+|016|Pension Benefit Guaranty Corporation|PBGC|1602|
+|033|Smithsonian Institution|SI|3300|
+|033|John F. Kennedy Center For The Performing Arts|Kennedy Center|3301|
+|033|National Gallery of Art|	National Gallery|3302|
+|033|Woodrow Wilson International Center For Scholars|Wilson Center|3303|
+|352|Farm Credit Administration|FCA|7801|
+|352|Farm Credit System Insurance Corporation|FCSIC|7802|
+|537|Federal Housing Finance Agency|FHFA|9566|
+|537|Federal Housing Finance Agency Inspector General|FHFAIG|9573|
+
+
+## Broker API Project Layout
+
+The Broker API has two major directories: scripts and handlers.
+
+```
+dataactbroker/
+├── scripts/        (Install and setup scripts)
+└── handlers/       (Route handlers)
+```
+
+### Scripts
+The `/dataactbroker/scripts` folder contains the install scripts needed to setup the broker API for a local install. For complete instructions on running your own copy of the API and other DATA Act broker components, please refer to the [documentation in the DATA Act core repository](https://github.com/fedspendingtransparency/data-act-broker-backend/blob/master/doc/INSTALL.md "DATA Act broker installation guide").
+
+### Handlers
+The `dataactbroker/handlers` folder contains the logic to handle requests that are dispatched from the `domain_routes.py`, `file_routes.py`, `login_routes.py`, and `user_routes.py` files. Routes defined in these files may include the `@requires_login` and `@requires_submission_perms` tags to the route definition. This tag adds a wrapper that checks if there exists a session for the current user and if the user is logged in, as well as checking the user's permissions to determine if the user has access to this route. If user is not logged in to the system or does not have access to the route, a 401 HTTP error will be returned. This tags are defined in `dataactbroker/permissions.py`.
+
+`account_handler.py` contains the functions to check logins and to log users out.
+
+`fileHandler.py` contains functions for managing user file interaction. It creates all of the jobs that are part of the user submission and has query methods to get the status of a submission. In addition, this class creates downloadable links to error reports created by the DATA Act Validator.
+
+## DATA Act Broker Route Documentation
+
+All routes that require a login should be passed a header "x-session-id".The value for this header should be taken from the login route response header "x-session-id".
+
+Route documentation is split into folders that reflect the internal organization of routes within the codebase. Each of these sections is detailed below, including a general description of the kinds of routes in each folder and links to documentation for each route in the folder.
+
+**All routes have an optional trailing slash, meaning the route will work with or without it.**
+
+### GET "/"
+This route is simply a check to ensure that Broker is running.
+
+Response: "Broker is running"
+
+### Dashboard Routes
```

Comment:
Love the way it's currently structured: table of contents that can delve into each section which then can be delved into each endpoint. Some part of me wants to go even further and move the lists into their associated directories so we don't have a giant list of endpoints out here but at that point it's just pointing to the files already in the folder. Maybe if we change the routing/directory strucutres we can explore this but for now this is fine.